csharp.test/TestWriterProperties.cs

```diff
@@ -112,7 +112,7 @@ public static void TestOverrideDefaults()
                 DefaultWriterProperties.MaxRowGroupLength = 789;
                 DefaultWriterProperties.Version = ParquetVersion.PARQUET_1_0;
                 DefaultWriterProperties.WriteBatchSize = 666;
-                DefaultWriterProperties.WritePageIndex = true;
+                DefaultWriterProperties.WritePageIndex = false;
```

Comment:
Is this change necessary?

Reply:
Same as above, this is changing the test to set the value to something different to the default, which changed from false to true.